src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Overview/Cells/DiaryOverviewDayTableViewCell.swift

```diff
@@ -20,6 +20,33 @@ class DiaryOverviewDayTableViewCell: UITableViewCell {
 		exposureHistoryDetailLabel.style = .subheadline
 		exposureHistoryDetailLabel.textColor = .enaColor(for: .textPrimary2)
 
+		// Check-Ins with risk
+		checkinHistoryStackView.isHidden = cellViewModel.hideCheckinRisk
+		checkinHistoryNoticeImageView.image = cellViewModel.checkinImage
+		checkinHistoryTitleLabel.text = cellViewModel.checkinTitleHeadlineText
+		checkinHistoryTitleLabel.accessibilityIdentifier = cellViewModel.checkinTitleAccessibilityIdentifier
+		checkinHistoryTitleLabel.style = .body
+		checkinHistoryDetailLabel.text = cellViewModel.checkinDetailDescription
+		checkinHistoryDetailLabel.style = .subheadline
+		checkinHistoryDetailLabel.textColor = .enaColor(for: .textPrimary2)
+		
+		checkinsWithRiskStackView.arrangedSubviews.forEach { $0.removeFromSuperview() }
+		
+		cellViewModel.checkinsWithRisk.forEach { riskyCheckin in
+			let checkInLabel = ENALabel()
+			checkInLabel.adjustsFontForContentSizeCategory = true
+			checkInLabel.numberOfLines = 0
+			checkInLabel.lineBreakMode = .byClipping
```

Comment:
Is this the correct line break mode for `numberOfLines = 0`?

Reply:
copy&paste :) I removed it.